packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -70,6 +70,8 @@ type Value<'scope> = v8::Local<'scope, v8::Value>;
 type Function<'scope> = v8::Local<'scope, v8::Function>;
 type Array<'scope> = v8::Local<'scope, v8::Array>;
 
+const MB: usize = 1_000_000;
```

Comment:
Do we perhaps want to use `1,048,576`? I genuinely don't know if it will matter at all, just wanted to know if using a power of 2 would be better here

Reply:
Changed in https://github.com/hashintel/hash/pull/446/commits/57ebc883dca4fcf964a9f86434b7bf6079a3ecb2